addons/sale_loyalty/tests/common.py

```diff
@@ -212,6 +212,8 @@ def _apply_promo_code(self, order, code, no_reward_fail=True):
             status = order._apply_program_reward(rewards, coupons)
             if 'error' in status:
                 raise ValidationError(status['error'])
+        elif len(coupons) == 1 and len(rewards) > 1:
+            return rewards
```

Comment:
What is this one for? Because when I checked for the program with multiple rewards it was still 1.